DuckDuckGo/AppUserDefaults.swift

```diff
@@ -202,4 +211,67 @@ extension AppUserDefaults: AppConfigurationFetchStatistics {
             userDefaults?.setValue(newValue, forKey: Keys.backgroundFetchNewDataCount)
         }
     }
+
+    var backgroundTaskExpirationCount: Int {
+        get {
+            return userDefaults?.integer(forKey: Keys.backgroundFetchTaskExpirationCount) ?? 0
+        }
+        set {
+            userDefaults?.setValue(newValue, forKey: Keys.backgroundFetchTaskExpirationCount)
+        }
+    }
```

Comment:
Can we use property wrappers here? Wouldn't that simplify the code?

Reply:
Great call, we should do that. I added it this way for consistency with the current approach, but new additions should definitely be using the more modern approach – I'll update this, thanks!